corehq/apps/locations/static/locations/ko/location_types.js

```diff
@@ -161,3 +163,8 @@ ko.bindingHandlers.bind_element = {
         }
     }
 };
+
+    return {
```

Comment:
That's a good point, and it sounds like it'd work.  I'm hesitant though, because I've been thinking of this in terms of modules, where the object returned is a namespace.

I could probably be talked out of it, but I'm inclined to make the define function always return a module, just for consistency.